consensus/state_sync/module.go

```diff
@@ -1,18 +1,39 @@
 package state_sync
 
 import (
+	"fmt"
+	"sync"
+	"time"
+
+	typesCons "github.com/pokt-network/pocket/consensus/types"
 	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/shared/codec"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/messaging"
 	"github.com/pokt-network/pocket/shared/modules"
+	"google.golang.org/protobuf/types/known/anypb"
 )
 
 const (
-	stateSyncModuleName = "stateSyncModule"
+	stateSyncModuleName       = "stateSyncModule"
+	committedBlocsChannelSize = 100
+	blockWaitingPeriod        = 30 * time.Second
+	metadataChannelSize       = 1000
 )
 
 type StateSyncModule interface {
 	modules.Module
 	StateSyncServerModule
+
+	HandleStateSyncBlockCommittedEvent(message *anypb.Any) error
+	HandleStateSyncMetadataResponse(metadataRes *typesCons.StateSyncMetadataResponse)
+
+	// SetActiveSyncHeight sets the active sync height
+	SetActiveSyncHeight(height uint64)
+
+	// StartActiveSync starts active state sync from current height to activeSyncHeight
+	StartActiveSync()
```

Comment:
Wouldn't it be simpler if we consolidate it with `StartActiveSync(height uint64)`?